src/components/Icon/icons/ChartBars2.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import iconColorsMap from './colors';
+
+const ChartBars2 = ({ fill, ...props }) => (
```

Comment:
Wondering if we should just call it `ChartBars` if we're not also importing the presumed original `ChartBars` ?

Reply:
Or maybe name it descriptively like the others

Reply:
I was just going off the names given to me in the svg file, but i agree that a few of them could have better names and happy to change any.

Reply:
I second @chelshaw ! Otherwise looks good.